llms/llama/convert.py

```diff
@@ -140,6 +139,21 @@ def quantize(weights, config, args):
     return quantized_weights, quantized_config
 
 
+def make_shards(weights: dict, max_file_size_GiB: int = 15):
+    max_file_size_bytes = max_file_size_GiB << 30
+    shards = []
+    shard, shard_size = {}, 0
+    for k, v in weights.items():
+        estimated_size = len(v.flatten()) * v.dtype.itemsize
```

Comment:
Did you check this with quantization? I think this line might break as `dtype` doesn't have an `itemsize`?

We really ought to expose `nbytes` in python.. for consistency with numpy. For now you can do:

```
v.size * v.dtype.size if isintance(v, mx.array) else v.nbytes
```

Reply:
I wasn't aware that quantization stores mx arrays already... 
your suggestion seems to be a good intermediate solution. Exposing `nbytes` sounds even better, I'll create a PR, sounds like a small change.